packages/extension/src/companion/index.tsx

```diff
@@ -28,7 +28,7 @@ browser.runtime.onMessage.addListener(
     visit,
     accessToken,
   }) => {
-    if (settings.optOutCompanion) {
+    if (!settings || settings?.optOutCompanion) {
```

Comment:
Why are we checking for `!settings`? is it still usable by anonymous users?

Reply:
Yes. Our anonymous users store cache data for their settings which are still being utilized. Also, boot data returns the default value for the anonymous one.

Reference snippet:
https://github.com/dailydotdev/apps/blob/af42b78916c8ed4d2dd782450e11bc1ac341141f/packages/extension/src/background/index.ts#L55-L58